src/Altinn.FileScan/Repository/Interfaces/IAppOwnerBlob.cs

```diff
@@ -1,4 +1,7 @@
-﻿namespace Altinn.FileScan.Repository.Interfaces
+﻿using Altinn.FileScan.Models;
+using Azure.Storage.Blobs.Models;
```

Comment:
Hmm, do you reference to Azure.Storage here?

Reply:
A drive by reference. Taken out.